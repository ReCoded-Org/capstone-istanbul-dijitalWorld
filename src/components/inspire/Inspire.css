.inspireContainer {
  text-align: center;
  margin: 0 15%;
}

.inspireHeader {
  font-size: 2.875rem;
}

.pinkInspire {
  color: #e87a80;
}

.purpleInspire {
  color: #6c567c;
}

.heroText {
  color: #4a474d;
  font-size: 1.875rem;
}

.inspireButton {
  background-color: #6c567b;
  color: white;
  cursor: pointer;
  width: 14rem;
  height: 4rem;
  font-size: 3rem;
  border-radius: 3.125rem;
  border-style: hidden;
  box-shadow: 0 0.6875rem 1.5rem -0.4375rem #b3b3b3;
  margin-bottom: 2%;
}

.inspireButton:hover {
  background-color: #f8b195;
}
