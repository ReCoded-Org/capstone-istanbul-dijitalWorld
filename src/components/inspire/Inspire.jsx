import React from "react";
import "./Inspire.css";
import Button from "react-bootstrap/Button";

export default function Inspire(props) {
  return (
    <div className="inspireContainer" data-testid="inspireContainer">
      <h2 className="inspireHeader">
        <span className="purpleInspire">{props.header0}</span>{" "}
        <span className="pinkInspire">{props.header1}</span>
        <span className="purpleInspire">{props.header2}</span>{" "}
        <span className="pinkInspire">{props.header3}</span>
      </h2>
      <p className="heroText">
        {props.heroText} <br />
      </p>
      <Button className="inspireButton">{props.button}</Button>{" "}
    </div>
  );
}
