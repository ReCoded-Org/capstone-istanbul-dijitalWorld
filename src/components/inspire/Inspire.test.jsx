import React from 'react';
import Inspire from './Inspire';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<Inspire />).toJSON();
  expect(tree).toMatchSnapshot();
});
