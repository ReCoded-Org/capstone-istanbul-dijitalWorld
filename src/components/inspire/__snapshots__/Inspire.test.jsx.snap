// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="inspireContainer"
  data-testid="inspireContainer"
>
  <h2
    className="inspireHeader"
  >
    <span
      className="purpleInspire"
    />
     
    <span
      className="pinkInspire"
    />
    <span
      className="purpleInspire"
    />
     
    <span
      className="pinkInspire"
    />
  </h2>
  <p
    className="heroText"
  >
     
    <br />
  </p>
  <button
    className="inspireButton btn btn-primary"
    disabled={false}
    type="button"
  />
   
</div>
`;
